BankSDK/GiniBankSDKExample/GiniBankSDKExampleUITests/GiniCameraAccessScreenUITests.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  GiniCameraAccessScreenUITests.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+import XCTest
+
+class GiniCameraAccessScreenUITests: GiniBankSDKExampleUITests {
+    
+    /*
+     On iOS <13.5 tests works only if app was deleted before launch or camera access was denied
+     */
+    
+    func testCameraAccessScreenBackButton() throws {
+        
+        //Reset Camera Access
+        app.terminate()
+    //Don't work on simulators
```

Comment:
[request] Please fix the indentation here.

Reply:
Fixed